src/commands/protocols/get/sender/local-get-command.js

```diff
@@ -94,22 +99,52 @@ class LocalGetCommand extends Command {
             operationId,
             blockchain,
         );
-        const assertionPromise = this.tripleStoreService
-            .getAssertion(
-                blockchain,
-                contract,
-                knowledgeCollectionId,
-                knowledgeAssetId,
-                contentType,
-            )
-            .then((result) => {
-                this.operationIdService.emitChangeEvent(
-                    OPERATION_ID_STATUS.GET.GET_LOCAL_GET_ASSERTION_END,
-                    operationId,
-                    blockchain,
-                );
+
+        let assertionPromise;
+
+        if (assertionId) {
+            assertionPromise = (async () => {
```

Comment:
It's a bit weird to put it in promise, maybe move it out of promise or create a separate promise for each repo. At least as TODO comment

Reply:
Also it's missing await